osutil/mountinfo_linux_test.go

```diff
@@ -159,20 +157,3 @@ func (s *mountinfoSuite) TestReadMountInfo2(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(entries, HasLen, 3)
 }
-
-// Test that loading mountinfo from a file works as expected.
```

Comment:
we probably want to keep these tests in some form using an in-package only export_test.go based mechanism?

Reply:
hmm yeah I can make that work, but not for the other place I had to remove those tests in selinux because it's a different package unfortunately.

Reply:
done